src/activities/components/ActivityItem.vue

```diff
@@ -26,6 +26,11 @@
               {{ $d(activity.date, 'dateLongWithDayName') }}
             </template>
           </div>
+          <ActivityEditButton
+            v-if="!preview && isEditor && !hasStarted"
```

Comment:
Ah, I see this does hide it if it's not preview, I was looking at the branch deployment, which maybe didn't deploy? ... I definitely saw it somewhere on a preview (but can't recreate it right now) :shrug:

Reply:
Does it branch deploy even when tests fail? Otherwise it might have been an older version.. on the other hand, the `!preview` condition has been there for some days already :)

Reply:
Yeah, not sure. I think we can assume it works ok, and if we notice it in the future can deal with it then...